template/docroot/sites/default/settings/files.settings.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/**
+ * @file
+ * Contains Files configuration.
+ */
+if ($is_acsf) {
+  $settings['file_private_path'] = "/mnt/gfs/$ah_group.$ah_env/files-private/$acsf_db_name";
+}
+else {
+  $settings['file_private_path'] = "/mnt/files/$ah_group.$ah_env/files-private";
```

Comment:
I think this is another place where we should consider multisite. D7 version would look something like `/mnt/files/' . $_ENV['AH_SITE_NAME'] . '/' . conf_path() . '/files-private'`, need to test the exact D8 syntax, but https://www.drupal.org/node/2275139 has some background since conf_path() is deprecated.